File: Codechef/Play_Fit/Play_Fit.py
T = int(input())
while T:
    T -= 1
    N = int(input())
    L = list(map(int, input().split()))
    # maximum = -1
    # # matrix = [[maximum for j in range(i + 1, N)] for i in range(N - 1) if L[i + 1] - L[i] > maximum]
    # for i in range(N):
    #     for j in range(i + 1, N):
    #         diff = L[j] - L[i]
    #         # print(diff)
    #         if (diff > maximum):
    #             maximum = diff
    #             # print("YES")
    # if maximum > 0:
    #     print(maximum)
    # else:
    #     print("UNFIT")

    minimum = L[0]
    maximum = 0
    for i in range(N):
        if L[i] < minimum:
            minimum = L[i]
        elif L[i] - minimum > maximum:
            maximum = L[i] - minimum
    if maximum > 0:
        print(maximum)
    else:
        print("UNFIT")

File: Codechef/Play_Fit/Play_Fit.java
/* package codechef; // don't place package name! */

import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
class Play_Fit
{
	public static void main (String[] args) throws java.lang.Exception
	{
		Scanner sc = new Scanner(System.in);
        int T = sc.nextInt();
        while (T-- > 0)
        {
            int N = sc.nextInt();
            int[] L = new int[N];
            for (int i = 0; i < N; i++)
            {
                L[i] = sc.nextInt();
            }
            int minimum = L[0], maximum = 0;
            for (int i = 0; i < N; i++)
            {
                if (L[i] < minimum)
                    minimum = L[i];
                else if (L[i] - minimum > maximum)
                    maximum = L[i] - minimum;
            }
            if (maximum > 0)
                System.out.println(maximum);
            else
                System.out.println("UNFIT");

        }
	}
}
